src/main/kotlin/dev/znci/rocket/commands/RocketCommand.kt

```diff
@@ -41,7 +41,7 @@ class RocketCommand(private val plugin: JavaPlugin) : TabExecutor {
 
         when (action) {
             "reload" -> {
-                if (scriptName.lowercase() == "config") {
+                if (scriptName.lowercase() == "config.lua") {
```

Comment:
Where is scriptName originally defined? I would expect `config` to be the correct name instead of `config.lua`.

Reply:
This is just a temporary fix for an issue that @Nuutrai will hopefully solve on their branch